data/manifest_core.json

```diff
@@ -5,55 +5,231 @@
         "subtype": {
           "h5n1": {
             "segment": {
-              "ha": "",
-              "mp": "",
-              "na": "",
-              "ns": "",
-              "np": "",
-              "pa": "",
-              "pb1": "",
-              "pb2": "",
+              "ha": {
+                "resolution": {
+                  "all-time": "",
+                  "2y": "",
+                  "default": "2y"
+                }
+              },
+              "pb2": {
+                "resolution": {
+                  "all-time": "",
+                  "2y": "",
+                  "default": "2y"
+                }
```

Comment:
The ordering of segments is different with this change. Instead of being alphabetical (ha, mp, na, …) it's now something else (ha, pb2, pb1, …). Was that intentional? (It wasn't called out in the commit message, so not clear to me.)

Reply:
Since this affects the order of the dropdowns, I'd think we'd want to keep the previous alphabetical ordering. (That's also what we force on https://nextstrain.org/pathogens.)

Reply:
[Restored](https://github.com/nextstrain/nextstrain.org/compare/297712220408e207adf7552a764eb53ecdf639c8..d899712136387b5628a7b785c30dbf33c13654d4) the not-quite-alphabetical order to:

```
ha
mp
na
ns
np
pa
pb1
pb2
```

Reply:
Yes. Let's keep the original ordering.

Reply:
This was me trying to make sure I didn't forget a particular segment by using flu segment ordering, which goes by longest to shortest. I don't have a preference on ordering at all!